clang/include/clang/Format/Format.h

```diff
@@ -4335,17 +4350,15 @@ struct FormatStyle {
           InEmptyParentheses(false), Other(false) {}
 
     SpacesInParensCustom(bool InConditionalStatements, bool InCStyleCasts,
-        bool InEmptyParentheses, bool Other)
+                         bool InEmptyParentheses, bool Other)
         : InConditionalStatements(InConditionalStatements),
-          InCStyleCasts(InCStyleCasts),
-          InEmptyParentheses(InEmptyParentheses),
+          InCStyleCasts(InCStyleCasts), InEmptyParentheses(InEmptyParentheses),
```

Comment:
This is an unrealted change, did you git clang-format?

Reply:
I ran the clang-format, and I am aware of this as I was waiting for the changes which I made and with the new commit, I will restore this.